.pre-commit-config.yaml

```diff
@@ -41,13 +41,15 @@ repos:
         args:
           - --rcfile=.pylintrc
 
-  # https://pre-commit.com/#repository-local-hooks
-  - repo: local
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.991
     hooks:
       - id: mypy
-        name: mypy
-        entry: mypy
-        language: system
-        types: [python]
-        files: \.py$
         exclude: ^setup.py|^tests/|^vendor/|^astacus/proto/
+        additional_dependencies:
+          - types-PyYAML>=6.0.12.2
+          - types-requests>=2.28.11.5
+          - types-tabulate>=0.9.0.0
+          - types-urllib3>=1.26.25.4
+          - typing_extensions>=3.10.0
+          - pydantic<2.0.0,>=1.7.4
```

Comment:
This should be 1.7.4 (f35) and whatever (once we get to f37).

Reply:
Fixing the constraint

Reply:
Pinning pydantic to `1.7.4` clashes with `plugins = pydantic.mypy` in `mypy.ini` due to a breaking change in mypy `0.920` (see https://github.com/pydantic/pydantic/issues/3528). The first pydantic version that fixes this is `1.9.1`.
IMO pinning mypy to `0.910` seems the most reasonable thing to do here.